docs/why.qmd

```diff
@@ -28,43 +317,59 @@ and robust framework for data manipulation in Python.
+formats. You can scale up with DuckDB (the default backend) or choose from other
+great options like Polars and DataFusion to work locally with large datasets.
+Once you hit scaling issues on a local machine, you can continue scaling up with
+a larger machine in the cloud using the same backend and same code.
+
+If you hit scaling issues on a large single-node machine, you can switch to a
+distributed backend like PySpark, BigQuery, or Trino by simply changing your
+connection string.
+
+## Stream-batch unification
+
+As of [Ibis 8.0](./posts/ibis-version-8.0.0-release/index.qmd), the first stream
+processing backends have been added. Since these systems tend to support SQL, we
+can with minimal changes to Ibis support both batch and streaming workloads with
+a single API. We aim to further unify the batch and streaming paradigms going
+forward.
+
 ## Ecosystem
 
 Ibis is part of a larger ecosystem of Python data tools.
 
 It is designed to work well with other tools in this ecosystem, and we continue
 to make it easier to use Ibis with other tools over time.
 
-### Local experience
-
-Out of the box, Ibis offers a great local experience for working with many file
-formats.
-
-DuckDB is the default backend, with Polars and DataFusion as two other great
-local options. Many of the backends can run locally but require more setup than
-a pip installation.
-
-### Scaling up and out
+Ibis already works with other Python dataframes like:
 
-After prototyping on a local backend, directly scale in the cloud.
+- [pandas](https://github.com/pandas-dev/pandas),
+- [Dask](https://github.com/dask/dask)
+- [Polars](https://github.com/pola-rs/polars)
 
-You can prototype on DuckDB and deploy with MotherDuck. You can scale from any
-Python client with Ibis installed to whatever your backend supports.
+Ibis already works well with visualization libraries like:
 
-## Use cases
+- [matplotlib](https://github.com/matplotlib/matplotlib)
+- [seaborn](https://github.com/mwaskom/seaborn)
+- [plotly](https://github.com/plotly/plotly.py)
+- [altair](https://github.com/altair-viz/altair)
+- [plotnine](https://github.com/has2k1/plotnine)
 
-You can use Ibis at any stage of your data workflow.
+Ibis already works well with dashboarding libraries like:
 
-Use the same framework for local exploration on a few files or production
-workloads on the most advanced data platforms.
+- [Streamlit](https://github.com/streamlit/streamlit)
+- [dash](https://github.com/plotly/dash)
+- [Quarto dashboards](https://github.com/quarto-dev/quarto-cli)
 
-Ibis helps with:
+Ibis already works well with machine learning libraries like:
 
-- data catalog exploration
-- exploratory data analysis
-- transforming data
-- visualizing data
-- data science and machine learning
+- [scikit-learn](https://github.com/scikit-learn/scikit-learn)
+- [XGBoost](https://github.com/dmlc/xgboost)
+- [LightGBM](https://github.com/microsoft/lightgbm)
+- [PyTorch](https://github.com/pytorch/pytorch)
```

Comment:
Broader documentation comment -- for everything like this that we advertise, I think we should aim to have short how-to docs on using X with Ibis.